studio/components/interfaces/Billing/PlanSelection/PlanSelection.tsx

```diff
@@ -16,11 +17,7 @@ interface Props {
 const PlanSelection: FC<Props> = ({ visible, tiers, currentPlan, onSelectPlan }) => {
   const formattedTiers = formatTierOptions(tiers)
 
-  // [Kevin] TODO Remove after team plan is generally available
-  const gridCols =
-    formattedTiers.length === 3
-      ? 'gap-8 lg:grid-cols-3'
-      : 'gap-4 lg:grid-cols-3 xl:grid-cols-billingWithTeam'
+  const enterprisePlan = formattedTiers.find((it) => it.name === 'Enterprise')!
```

Comment:
=== `tierNameVar` rather than `"Enterprise"` ?

if plan name changes this will break.